Doc/library/dbm.rst

```diff
@@ -164,6 +162,8 @@ SQLite backend for the :mod:`dbm` module.
 The files created by :mod:`dbm.sqlite3` can thus be opened by :mod:`sqlite3`,
 or any other SQLite browser, including the SQLite CLI.
 
+.. include:: ../includes/wasm-notavail.rst
+
```

Comment:
I'd remove these -- the top-level one indicates that the entire `dbm` package isn't supported.

```suggestion
```

Reply:
I removed the top-level notice because it appears to be incorrect: the WASI buildbot is passing the `dbm.dumb` test, and the iOS buildbot is passing `dbm.dumb` and `dbm.sqlite3`. So we need separate notices on each submodule.